server/sonar-server/src/main/java/org/sonar/server/component/es/ProjectMeasuresIndexDefinition.java

```diff
@@ -34,6 +36,12 @@
   public static final String FIELD_MEASURES_KEY = "key";
   public static final String FIELD_MEASURES_VALUE = "value";
 
+  public static final String TYPE_AUTHORIZATION = "authorization";
+  public static final String FIELD_AUTHORIZATION_PROJECT_UUID = "project";
+  public static final String FIELD_AUTHORIZATION_GROUPS = "groups";
+  public static final String FIELD_AUTHORIZATION_USERS = "users";
```

Comment:
This field contains logins. I suggest to convert to user ids as functional keys should be avoided in backend. In that case the name "users" is good. If you keep logins, then field should be renamed "userLogins".